### PR TITLE
DDO-1252 Adding prometheus-to-sd deployment for Kube State Metrics

### PR DESCRIPTION
**Pull request by mflinn-broad**

### CODE REVIEW

File: charts/terra-prometheus/values.yaml [hunk 1]
```diff
@@ -41,7 +41,19 @@ vaultCert:
     # vaultCert.chain.secretKey -- (string) Key in secret containing intermediate .crt
     secretKey: 
 
-
+# prometheus-to-sd values. Used specifically for sending kube state metrics to sd
+prometheusToSD:
+  # prometheusToSD.imageRepo -- (string) image repo for prometheus-to-sd-exporter
+  imageRepo: gcr.io/google-containers/prometheus-to-sd
+  # prometheusToSD.imageRepo -- (string) image tag for prometheus-to-sd-exporter
```
Comment (choover-broad) [inline charts/terra-prometheus/values.yaml:48]: typo, should be `imageTag`?

**Review by choover-broad — APPROVED** [review]
Awesome! 🎉 

Wondering if `deployment.yaml` and `serviceAccount.yaml` should be called something more specific or live under a subdirectory? Eg. `prometheus-to-sd/deployment.yaml`?

File: charts/terra-prometheus/values.yaml (hunk 1 above)
Comment (mflinn-broad) [reply charts/terra-prometheus/values.yaml:48]: yep, thanks

**Comment by mflinn-broad** [comment]
> Awesome! 🎉
> 
> Wondering if `deployment.yaml` and `serviceAccount.yaml` should be called something more specific or live under a subdirectory? Eg. `prometheus-to-sd/deployment.yaml`?

Good call, will do!

**Pull request merged**